front/src/reducers/infra/index.ts

```diff
@@ -2,20 +2,26 @@ import type { CaseReducer, Draft, PayloadAction } from '@reduxjs/toolkit';
 
 export interface InfraState {
   infraID: number | undefined;
+  infraLockStatus: boolean;
 }
 
 export const infraState: InfraState = {
   infraID: undefined,
+  infraLockStatus: false,
```

Comment:
I think infraLockStatus is a very good name (true, false, what does it mean ?). infraIsLocked seems better, but maybe you can find smth even better ;)
```suggestion
  infraIsLocked: false,
```

Reply:
fixed